contracts/ens/ens.go

```diff
@@ -1,5 +1,21 @@
 //go:generate abigen --sol contract/ENS.sol --pkg contract --out contract/ens.go
```

Comment:
Please move this line below the package declaration. I should've said that earlier, sorry.